### DISCUSSION TITLE
HTTPS Requirement

### DISCUSSION THREAD

**Discussion by elronzo** [house-of-abbey/GarminHomeAssistant#188] [Q&A]
Is providing HomeAssistant via SSL enough or do I also have to expose it to the Internet?

**Comment by philipabbey** [comment]
SSL should be enough. Not exposing to the Internet requires some effort, e.g DNS, and we cannot support you for that.

**Reply by yulrizka** [reply]
First of all good work on the project. 

Would it work if I just want to be able to access it from local network? why does self signed certificate does not work?

**Reply by philipabbey** [reply]
Due to the Garmin SDK, over which we have no control. Its HTTPS or nothing. Make your choice! There has to be a chain of trust, hence we suggest Let's Encrypt.
